src/components/user/repo/classes.repo.js

```diff
@@ -0,0 +1,25 @@
+const getService = (app) => {
+	return app.service('classModel');
+};
+
+const getClasses = async (userId, app) => {
+	const classes = await getService(app).find({
+		query: {
+			userIds: { $in: userId },
+		},
+		paginate: false,
+	});
+	return Array.isArray(classes) ? classes : [classes];
```

Comment:
Can the result be an array OR single object?
Accordingly to this docs https://docs.feathersjs.com/api/databases/common.html#adapter-find-params its always an array.
But you're right, you should always return an array.